src/module-elasticsuite-catalog/Model/Layer/Filter/Category.php

```diff
@@ -231,7 +241,7 @@ protected function getChildrenCategories()
      */
     protected function getFilterField()
     {
-        return 'category.category_id';
+        return $this->requestFieldMapper->getMappedFieldName('categories');
```

Comment:
Why not keeping "category_id" here ? This could ensure we have a difference between legacy category filter and the virtual one.

Reply:
No need to keep "category_id". 
To keep consistency the actual value that should be is the attribute code which stores product categories, which is "category_ids'.

Changed to: `$this->requestFieldMapper->getMappedFieldName('category_ids');`